src/main/lrsql/admin/interceptors/ui.clj

```diff
@@ -9,20 +9,26 @@
   [_]
   (resp/redirect "/admin/index.html"))
 
-(def get-env
-  "Provide select config data to client upon request"
+(defn get-env
+  "Provide select config data to client upon request. Takes a map with static
+  config to inject:
+    :enable-admin-status - boolean, determines if the admin status endpoint is
+      enabled."
+  [{:keys [enable-admin-status]
+    :or   {enable-admin-status false}}]
```

Comment:
More broadly, is there a reason why you're passing in `enable-admin-status` as a function arg, instead of pulling it directly from `ctx` like the existing config values?

Reply:
I made that choice because it seemed better to do all the work at init (since it turns off a route) rather than adding another interceptor just to stick a value in at runtime. If there was a single interceptor for sticking static config into the CTX at runtime I might have made a different call, but I couldn't find one so I chose this.

Reply:
I wonder how all the other env vars in that function/interceptor _do_ get injected into the context then. Feels weird that we'd introduce some on init and others at runtime.

Reply:
Two of them come from the LRS lib (path-prefix and statement-html?) and are added for runtime because they are used in a variety of places. The OIDC env is added by an interceptor because it is also required at runtime I believe, but if it is just being injected for this it could be moved into a static arg (though that would be outside the scope of this PR)